Gemfile

```diff
@@ -5,3 +5,5 @@ gem 'puppet', puppetversion
 gem 'puppetlabs_spec_helper', '>= 0.1.0'
 gem 'puppet-lint', '>= 1.0.0'
 gem 'facter', '>= 1.7.0', "< 1.8.0"
+gem 'rspec-puppet', '~>1.0'
+gem 'rspec', '~>2.0'
```

Comment:
@jwennerberg: Hint: you'll need to add above two lines to successfully run spec tests for all rund and PRs. Newer gems are broken at the moment.